kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -524,6 +524,65 @@ def test_seed(self):
 
         ws.close()
 
+class TestSeedGatewayAppKernelLanguageSupport(TestGatewayAppBase):
+    def setup_app(self):
+        self.app.prespawn_count = 1
+        self.app.seed_uri = os.path.join(RESOURCES,
+                                         'zen2.ipynb')
+    @coroutine
+    def spawn_kernel(self):
+        ws = yield super(TestSeedGatewayAppKernelLanguageSupport, self).spawn_kernel(kernel_body='{\"name\":\"python2\"}')
+        return ws
+
+    @gen_test
+    def test_seed_kernel_name_language_support(self):
+        '''Python2 Kernel should have the correct name'''
+        _, kernel_id = yield self.app.kernel_pool.acquire()
+        kernel = self.app.kernel_manager.get_kernel(kernel_id)
+        self.assertEqual(kernel.kernel_name,'python2')
+
+    @gen_test
+    def test_seed_language_support(self):
+        '''Python2 Kernel should have variables preseeded from notebook. Failures may be the result of networking problems.'''
```

Comment:
This second test is not doing anything python2 specific. You can change the print statement below to be:
```
print this.s
```
This would verify the kernel is running python2. If your intent was not to check that the code is python2 then I would remove this tests because it is testing the same thing as the test in `TestSeedGatewatApp`.